jpg/metadata/Reconyx HC500 firmware 3.3.0.jpg.txt

```diff
@@ -0,0 +1,57 @@
+FILE: Reconyx HC500 firmware 3.3.0.jpg
+
+[JPEG - 0xfffffffd] Compression Type = Baseline
+[JPEG - 0x0000] Data Precision = 8 bits
+[JPEG - 0x0001] Image Height = 1536 pixels
+[JPEG - 0x0003] Image Width = 2048 pixels
+[JPEG - 0x0005] Number of Components = 3
+[JPEG - 0x0006] Component 1 = Y component: Quantization table 0, Sampling factors 2 horiz/1 vert
+[JPEG - 0x0007] Component 2 = Cb component: Quantization table 1, Sampling factors 1 horiz/1 vert
+[JPEG - 0x0008] Component 3 = Cr component: Quantization table 1, Sampling factors 1 horiz/1 vert
+
+[Exif IFD0 - 0x011a] X Resolution = 72 dots per inch
+[Exif IFD0 - 0x011b] Y Resolution = 72 dots per inch
+[Exif IFD0 - 0x0128] Resolution Unit = Inch
+[Exif IFD0 - 0x0213] YCbCr Positioning = Datum point
+
+[Exif SubIFD - 0x9000] Exif Version = 2.20
+[Exif SubIFD - 0x9101] Components Configuration = YCbCr
+[Exif SubIFD - 0xa000] FlashPix Version = 1.00
+[Exif SubIFD - 0xa001] Color Space = sRGB
+[Exif SubIFD - 0xa002] Exif Image Width = 2048 pixels
+[Exif SubIFD - 0xa003] Exif Image Height = 1536 pixels
+[Exif SubIFD - 0x8827] ISO Speed Ratings = 50
+[Exif SubIFD - 0x829a] Exposure Time = 1/60 sec
+
+[Reconyx Makernote - 0x0000] Makernote Version = 61697
+[Reconyx Makernote - 0x0002] Firmware Version = 3.3.0
+[Reconyx Makernote - 0x0008] Firmware Date = 8209 272
+[Reconyx Makernote - 0x000c] Trigger Mode = M
+[Reconyx Makernote - 0x000e] Sequence = 4/5
+[Reconyx Makernote - 0x0012] Event Number = 0 7
+[Reconyx Makernote - 0x0016] Date/Time Original = 2015:05:07 08:08:31
+[Reconyx Makernote - 0x0024] Moon Phase = Waning Gibbous
```

Comment:
Never would have guessed we'd end up with lunar metadata :) This looks like a very interesting camera.